nextflow.config

```diff
@@ -179,15 +184,18 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     docker {
-        conda.enabled = false
-        docker.enabled = true
-        singularity.enabled = false
-        podman.enabled = false
-        shifter.enabled = false
-        charliecloud.enabled = false
-        docker.runOptions = '-u $(id -u):$(id -g)'
+        docker.enabled         = true
+        // docker.registry        = 'quay.io'
```

Comment:
Is it still not working with the merged modules? :(

Reply:
ah ok got it just fixed that. So when "quay.io/" is present that does not break things. I was wondering this because none of the modules were yet updated in this respect, but it works like that